package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "bookstoreredux",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.7",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.1.3",
+    "react-router-dom": "^6.18.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
```

Comment:
- [ ] I was unable to access your app initially because some dependencies are missing from the package.json file, kindly use the `npm install` command to resolve that. Thanks. 👍 

`example npm install axios`

<img width="904" alt="6" src="https://github.com/samuelkinyanjui/bookstoreredux/assets/103386183/66ab9016-2bc5-4645-b095-9ec1f5302551">